wire/core/PagesEditor.php

```diff
@@ -398,6 +398,9 @@ public function setupPageName(Page $page, array $options = array()) {
 				if(strlen($page->title)) $pageName = $page->title;
 					else $pageName = $this->untitledPageName;
 
+			} elseif($format == 'id') {
```

Comment:
`elseif` > `else if`

Reply:
While I also prefer `elseif` it seems that @ryancramerdesign prefers `else if`. Do you have a reason for why one is actually better than the other?

Reply:
`elseif` is the recommended way per PSR-2
https://www.php-fig.org/psr/psr-2/#51-if-elseif-else

Reply:
While referring to PSR-2 is a reasonable argument, we haven't adopted that for ProcessWire. That being said, our own style guide recommends `else if` instead of `elseif` as well: https://processwire.com/api/coding-style-guide/#5.1-if-elseif-else :)